ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.ts

```diff
@@ -402,6 +401,40 @@ const triggerSet: TriggerSet<Data> = {
       netRegexKo: NetRegexes.startsUsing({ id: '5C7[34]', source: ['헨젤', '그레텔'], capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'Paradigm Hansel/Gretel Bloody Sweep',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5C5[4567]', source: ['Hansel', 'Gretel'] }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      durationSeconds: 5,
+      suppressSeconds: 1,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '5C54' || matches.id === '5C55') {
+          // Hansel is West and Gretel is East
+          if (parseFloat(matches.castTime) > 12) {
+            // Hansel and Gretel will switch places
+            return output.north!();
+          }
+          // Hansel and Gretel stay in same position
+          return output.south!();
+        }
+        // Gretel is West and Hansel is East
+        if (parseFloat(matches.castTime) > 12) {
+          // Hansel and Gretel will switch places
+          return output.south!();
+        }
+        // Hansel and Gretel stay in same position
+        return output.north!();
+      },
+      outputStrings: {
+        north: {
+          en: 'Go North',
```

Comment:
Do you think this needs to be "Go North" vs "North"? You could say `north: Outputs.north` here, or we could add a `goNorth` to `Outputs` as well?

Reply:
I think just saying "north" is fine.

Reply:
Alternatively, we just used "Go North" in the meng-zi/xun-zi fight so maybe that is a good idea too